ethereum/statetransition/src/main/java/tech/pegasys/teku/statetransition/block/BlockManager.java

```diff
@@ -138,14 +181,14 @@ private void importBlockIgnoringResult(final SignedBeaconBlock block) {
               } else {
                 switch (result.getFailureReason()) {
                   case UNKNOWN_PARENT:
-                    // Add to the pending pool so it is triggered once the parent is imported
+                    // Add to the pending pool so it is triggered once the parent is
+                    // imported
```

Comment:
Might exceed line limit but worth checking if we can rejoin this comment (sometimes it gets wrapped and "sticks" like that when it would fit to join).